docker/local_pudl_etl.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/bash
```

Comment:
A comment explaining when this script is meant to be used would be helpful. It seems like it's meant to run inside the Docker container, but on a local machine, and really only for the purpose of debugging the GCP deployment? Otherwise always bypassing the local cache seems like it would be really annoying and cost a bit in egress fees.

Reply:
Ah great point thank you. I removed --bypass-local-cache from `local_pudl_etl.sh`.

Reply:
I also added a comment descriptions to the two scripts.